docker/Dockerfile

```diff
@@ -100,10 +100,14 @@ ENV PYTHONUNBUFFERED=1 \
     prometheus_multiproc_dir=/prom_cache
 
 # Install all OS package upgrades and dependencies needed to run Nautobot in production
-# hadolint ignore=DL3005,DL3008,DL3013
+# hadolint ignore=DL3005,DL3008,DL3009,DL3013,DL4006
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install --no-install-recommends -y git mime-support curl libxml2 libmariadb3 openssl && \
+    apt-get install --no-install-recommends -y curl
+
+# hadolint ignore=DL3008,DL3009,DL3013,DL3059,DL4006
```

Comment:
Really wish hadolint worked like pylint and let you specify human-readable labels to ignore. What are all of these?

Reply:
I’ll throw in some comments